### PR TITLE
LPS-117120 - Fix global menu height in mobile

### PR DESCRIPTION
**Pull request by marcoscv-work**
This PR fixes a problem in mobiles (issue for iPhone7) to get to see all the global menu vertical space.

---

To test the PR you only need to open your portal in your mobile by using your Mac/PC local IP.

iphone7

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

File: modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss [hunk 1]
```diff
@@ -63,6 +63,14 @@ $tab-header-height: 38px;
 	margin: 0;
 	padding: 0.2rem;
 
+	@include media-breakpoint-down(xs) {
+		@at-root {
+			.mobile #{&} {
+				padding-bottom: 7rem;
```
Comment (jbalsas) [inline modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: bleblu

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: Just in case the gif is not clear... what does `.mobile #{&}` even mean? 😂 

Just out of curiosity :)

Comment (eduardoallegrini) [inline modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: I don't like the hardcoded value 7rem but, after a brief with @marcoscv-work, I understand that:
- it's a exclusive fix for mobile
- it's the value of the header that hardly will change
So I think is correct 😁

I have a question: do we intend to change this part with the drilldown-menu?

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: Probably, but not for 7.3 to avoid last-minute instability

Comment (eduardoallegrini) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: The `#{&}` stands for the first class in the section
```scss
.lorem {
   .ipsum {
        .mobile #{&} {...
```
Will be
```css
.lorem .ipsum .mobile .lorem {...
```
We use it to avoid repeating the name of the parent class but since we don't want the `.mobile` class to be a child of the others we add `@at-root` to exit the inheritance and obtain 
```css
.mobile .lorem {...
```

Comment (pat270) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: We don't need to interpolate (`#{}`) the Sass parent selector (`&`) here. We only need to interpolate when we want the selector `.mobile.global-apps-nav` (`.mobile#{&}`).

Personally, I don't like using `.selector &` or `@at-root` unless I have to (e.g., in a mixin). It's hard to follow.

```
.mobile .global-apps-nav {
  @include media-breakpoint-down(xs) {
    padding-bottom: 7rem;
  }
}
```
is easier to read IMO.

Comment (marcoscv-work) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: We will remove all of this is finally use drilldown

Comment (pat270) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: @eduardoallegrini brings up an interesting Sass feature?

|selector|output|
|---|---|
|`.mobile &`|`.mobile .global-apps-nav`|
|`.mobile #{&}`|`.global-apps-nav .mobile .global-apps-nav`|
|`@at-root { .mobile #{&} }`|`.mobile .global-apps-nav`|
|`.mobile#{&}`|`.global-apps-nav .mobile.global-apps-nav`|
|`@at-root { .mobile#{&} }`|`.mobile.global-apps-nav`|

Comment (eduardoallegrini) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: I think Marcos wanted to keep `.global-apps-nav` as the outer class and `.mobile` as "external modifier" 😆 (I don't mind)

Comment (eduardoallegrini) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: I thought about `.mobile &` too, but does it work with the mixin `@include media-breakpoint-down(xs) {` in the way?

Comment (pat270) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: It works, the only thing IIRC that doesn't work in media queries is `@extend`.

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: Thanks for the cookbook @pat270!

I still understand we're good to forward this? or do you guys want to make any changes?

Comment (jbalsas) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:69]: Wanna hit the forward button, then? 😉

**Comment by pat270** [comment]
ci:forward
